### ISSUE TITLE
Wrong version of binary at guthub?

### ISSUE THREAD

**Issue by ralski** [prest/prest#292]
- pREST version (or commit ref): 0.3.0 = expected, however.. read below
- pREST endpoint:
- PostgreSQL version: 9.5.10
- Operating system: CentOS 6, 32bit
- Go version: None.
- Log gist: 

## Description

Hello.

I have downloaded the binary from a section titled '0.3.0' of github, from this exact link: https://github.com/prest/prest/releases/download/v0.3.0/prest-linux-386

Then I run it with parameter 'version' and it says:
2018/03/22 18:46:47 [warning] You are running pREST in public mode.
2018/03/22 18:46:47 [warning] config.go:156 You are running pREST in debug mode.
Serve a RESTful API from any PostgreSQL database 0.1.7

So.. is it really a v0.3.0 build, just mis-labeled, or is it a v0.1.7 build mistakenly placed in 0.3.0 download section?

I am asking because I have a feeling, that it completely ignores the '[cors]' section of config file.

I am watching the communtication between Firefox, pREST and PG using ngrep:
  ngrep -d lo port 3000
and this suspicious executable does not sends back the 'Access-Control-Allow-Origin' header along with data from pg, no matter what I do store in '[cors]' section of config file.

**Comment by felipeweb** [comment]
cc @avelino

**Comment by felipeweb** [comment]
@ralski  just the binary prints the wrong version. Could you try with master to verify if the cors bug still occurs?

**Comment by ralski** [comment]
@Felipe

Sorry for late response, but I was without internet connection from April beginning until now.

Thank you for checking the version issue.

Regarding your recommendation: I have cloned the project using command `git clone https://github.com/prest/prest`, but I am not familiar with Go compiler. Standing in the project directory, I tried some quickly found recommendations from web, but it complained about missing importfile.
```
root@rig:/proj/prest # go tool compile main.go
main.go:4:2: can't find import: "github.com/prest/adapters/postgres"
```
Then I located the "github.com/prest/adapters/postgres" directory in subdir "vendor", so I have tried:
```
root@rig:/proj/prest # cd vendor/
root@rig:/proj/prest/vendor # go tool compile ../main.go 
../main.go:4:2: can't find import: "github.com/prest/adapters/postgres"
```
Well, am I doing some trivial mistake? I am not asking for an extensive tutorial on compiling a Go project. But if I really just miss some trivial step(s) to make it possible, I would be happy to get the help to be able to compile it myself.
